package.json

```diff
@@ -1,9 +1,12 @@
 {
   "name": "fss-node-metainspector",
   "description": "Npm package for web scraping purposes. You give it an URL, and it lets you easily get its title, links, images, description, keywords, meta tags",
-  "version": "1.3.1",
+  "version": "1.3.2",
   "homepage": "http://github.com/gabceb/node-metainspector",
   "repository": "git://github.com/gabceb/node-metainspector.git",
+  "publishConfig": {
```

Comment:
So where are we actually configuring the npm registry? I agree it shouldn't be configured in the code, but It doesn't seem to be currently configured in the Jenkins job.